picard/webservice/__init__.py

```diff
@@ -341,6 +350,20 @@ def __init__(self, parent=None):
         self._init_queues()
         self._init_timers()
 
+    def ssl_errors(self, reply, errors):
+        # According to forums, sometimes sslErrors is triggered with errors set to NoError
+        # This can also be used to ignore others if needed
+        ignored_errors = {
+            QSslError.NoError,
+        }
+        has_errors = False
+        for error in errors:
+            if error not in ignored_errors:
+                has_errors = True
+                log.error("SSL error: %s" % error.errorString())
+        if not has_errors:
```

Comment:
Would it be better to remove any NoError entries from errors, then you can do `if not errors: return` and then loop and do `ignoreSslErrors()`?

Reply:
I'm reluctant to modify the passed list, and it wouldn't be much an improvement over current code.